cpp/oneapi/dal/algo/jaccard/common.cpp

```diff
@@ -22,9 +22,9 @@ namespace jaccard {
 class detail::descriptor_impl : public base {
 public:
     std::int64_t row_begin    = 0;
-    std::int64_t row_end      = 1;
+    std::int64_t row_end      = 0;
```

Comment:
should not those names be aligned with external names?

Reply:
We support two APIs set_row_range, set_block_range or set_block. Current naming is consistent with first option and clear for 2. If we change inner layout and names to std init lists that will confuse users who use first: it would be bigger in code, we will need to create init list from two numbers which not always in seq memory meanwhile make two numbers from std list will be cheaper.

Reply:
there is no need in initializer lists in the implementation. The expectation - to have a single naming style which will help me quickly correlate api and implementation

Reply:
@andreynv 
row_begin -> row_range_begin?

Reply:
changed to row_range_begin, get_row_range_begin()